### PR TITLE
Submited version for review

### PR DESCRIPTION
**Pull request by NeckerFree**
Pull request created to practice Review process.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="./css/style.css">
+</head>
```
Comment (NeckerFree) [inline index.html:6]: Please include the Linters Validation and the required structure:
- include the language in the HTML tag.
- include the Meta tags inside the Head Tag.
- include The Title tag inside the Head Tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                            height="30px"></a><Label>   Birds</Label></li>
+            </ul>
+            <ul>
+                <li><a href="http://"></a><img src="./images/panda.jpg" alt="Animals" width="30px"
+                        height="30px"></a><Label>   Panda</Label></li>
+            </ul>
+            <ul>
+                <li><a href="http://"></a><img src="./images/squirrel.jpg" alt="Animals" width="30px"
+                        height="30px"></a><Label>   Squirrel</Label></li>
+            </ul>
```
Comment (NeckerFree) [inline index.html:32]: Please do not directly include attributes like width and height directly in the img tag, instead, it is better to use a CSS class.
It is not good practice to include these attributes within HTML.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                <h1>Pre-enrollment Video</h1>
+                <iframe src="https://www.youtube.com/embed/2IEvEXxQ2og" title="YouTube video player" frameborder="10"
+                    width=600px height=500px
+                    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                    allowfullscreen></iframe>
+                    <h1>Amaizing nature</h1>
+                    <p>Video about the different bio diversity biomas</p>
+                    <label>number of views: (100)</label>
+            </section>
+        </div>
```
Comment (NeckerFree) [inline index.html:46]: The use of the tag div is not necessary. Kindly consider removing it and use only the section to give this tag the CSS attributes required.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## Hi @NeckerFree 👋, 

Good job so far on this special project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

- [ ]  You performed three different code reviews for your request changes, please include all your inline comments in one review. After adding the first comment you can click on start review and you can add as many comments as you want within that single code review. After adding all comments with requested changes click on finish review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (NeckerFree) [inline index.html:6]: Please include the Linters Validation and the required structure:
-include the language in the HTML tag.
-include the Meta tags inside the Head Tag.
-include The Title tag inside the Head Tag.

File: index.html (hunk 2 above)
Comment (NeckerFree) [inline index.html:32]: Please do not directly include attributes like width and height directly in the img tag, instead, it is better to use a CSS class.
It is not good practice to include these attributes within HTML.

File: index.html (hunk 3 above)
Comment (NeckerFree) [inline index.html:46]: The use of the tag div is not necessary. Kindly consider removing it and use only the section to give this tag the CSS attributes required.

**Review by NeckerFree — COMMENTED** [review]
Hi @NeckerFree,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Required changes** ♻️ 
PLease review the comments 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: index.html (hunk 1 above)
Comment (nicupop729) [reply index.html:6]: When we are reviewing the code written by students, we want to deliver a clear and specific message. In this comment, you suggested to the student multiple requirements which are not related. As linter validation is not something that relates to a specific line of code, we should address it in the box where we tell to the student if the project is approved or not. :sun_behind_small_cloud: 

Related to the required structure of `html` tag and of `head` tag, we can include examples to be more specific, we can provide links. And always we should explain ourselves. Why do we suggest a particular thing to the student? :sun_with_face: 

[Here is a link about `head` tag](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/The_head_metadata_in_HTML) that we can provide to the student for more clarity. 

- [ ] Please update the comments to be more specific regarding the `html` and `head` tags and please provide useful links. :+1: 

- [ ] Kindly, remove the remark about linters from this comment but don't forget to include it in the general box, where we tell to the student if we approved his project or not. :sunglasses:

File: index.html (hunk 2 above)
Comment (nicupop729) [reply index.html:32]: - [X] Very well for using the magic word "Please", and very good that you brought explanations.

File: index.html (hunk 3 above)
Comment (nicupop729) [reply index.html:46]: - [X] Very good remark :1st_place_medal:

**Review by nicupop729 — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️

Hi @NeckerFree, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
stay-strong-youre-almost-there

   * Good understanding of OOP✔️

 ### Required Changes ♻️

- [ ]  _Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
```
Comment (NeckerFree) [inline index.html:2]: According to the Microverse Linters validation, the `lang` attribute is missing, please include it in the `HTML` tag.

Comment (NeckerFree) [inline index.html:2]: According to the Microverse Linters validation, the `lang` attribute is missing, please include it in the `html` tag.

File: index.html (hunk 1 above)
Comment (NeckerFree) [inline index.html:6]: In the `head` tag, the tags `meta` and `title` are required. please include them on your page.

Here is a link that explains the structure of the `head` tag: [link](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/The_head_metadata_in_HTML)

File: index.html (hunk 2 above)
Comment (NeckerFree) [inline index.html:32]: Please do not directly include attributes like `width` and `height` directly in the `img` tag, instead, it is better to use a CSS class.
It is not good practice to include these attributes within HTML.

File: index.html (hunk 3 above)
Comment (NeckerFree) [inline index.html:46]: In this portion of code, the use of the tag `div` is not necessary. Kindly consider removing it and use only the `section` to give this tag the CSS `class` attribute required.

**Review by NeckerFree — COMMENTED** [review]
Hi @NeckerFree,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Required changes ♻️:
Please include the Linters Validation and the required structure. Please follow this document to configure and the Linters: [link](https://github.com/microverseinc/linters-config/tree/master/html-css), and review below the additional comments.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Review by thecodechaser — APPROVED** [review]
## Hi @NeckerFree 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**